results/LaBSE/e34fab64a3011d2176c99545a93d5cbddc9a91b7/FaithDial.json

```diff
@@ -1,8 +1,8 @@
 {
   "dataset_revision": "7a414e80725eac766f2602676dc8b39f80b061e4",
-  "evaluation_time": 5.803838729858398,
```

Comment:
Why did these change again? You might have said before but I thought this was new results and Flores fixes?

Reply:
This was changed in #6 so I have just carried the changes here. I imagine @Muennighoff simply reran the models.